### PR TITLE
Lt 4116 follow up

### PR DESCRIPTION
**Pull request by gponomarev-lykke**

### CODE REVIEW

File: contract/Lykke.Snow.PriceAlerts.Contract/Models/Requests/GetPriceAlertsRequest.cs [hunk 1]
```diff
@@ -6,7 +8,9 @@ public class GetPriceAlertsRequest
     {
         public string ProductId { get; set; }
 
-        public AlertStatusContract[] Statuses { get; set; }
+        [CanBeNull]
```
Comment (tarurar) [inline contract/Lykke.Snow.PriceAlerts.Contract/Models/Requests/GetPriceAlertsRequest.cs:11]: CanBeNull or ItemCanBeNull?

File: src/Lykke.Snow.PriceAlerts.Domain/Extensions/DecimalExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+using System.Linq;
+
+namespace Lykke.Snow.PriceAlerts.Domain.Extensions
+{
+    public static class DecimalExtensions
+    {
+        private const int DefaultDigitPrecision = 5;
+        
+        public static string ToUiString(this decimal? value, int? precision = null)
```
Comment (tarurar) [inline src/Lykke.Snow.PriceAlerts.Domain/Extensions/DecimalExtensions.cs:9]: Not sure if Price Alerts service is the best match for UI transformations.Looks like it's main responsibility is price alerts processing whereas interaction with user should be on Meteor shoulders.

File: src/Lykke.Snow.PriceAlerts.Domain/Services/IMeteorService.cs [hunk 3]
```diff
@@ -3,8 +3,8 @@
 
 namespace Lykke.Snow.PriceAlerts.Domain.Services
 {
-    public interface IPriceAlertCqrsSender
+    public interface IMeteorService
```
Comment (tarurar) [inline src/Lykke.Snow.PriceAlerts.Domain/Services/IMeteorService.cs:6]: I think better match would be `MeteorSender`, not service.

**Review by tarurar — APPROVED** [review]

File: contract/Lykke.Snow.PriceAlerts.Contract/Models/Requests/GetPriceAlertsRequest.cs (hunk 1 above)
Comment (gponomarev-lykke) [reply contract/Lykke.Snow.PriceAlerts.Contract/Models/Requests/GetPriceAlertsRequest.cs:11]: I think canBeNull, we can skip sending an array and we will receive alerts with any status

File: src/Lykke.Snow.PriceAlerts.Domain/Extensions/DecimalExtensions.cs (hunk 2 above)
Comment (gponomarev-lykke) [reply src/Lykke.Snow.PriceAlerts.Domain/Extensions/DecimalExtensions.cs:9]: Alex implemented this part, also we need to reevaluate how accuracy and precision work on the platform https://lykke-snow.atlassian.net/browse/LT-4127?atlOrigin=eyJpIjoiYjNmOTcyYTZhNjE1NDdhYjg4YmQzMmU4NGNiZmM5Y2IiLCJwIjoiamlyYS1zbGFjay1pbnQifQ
I think we will be able to fix it after this task is done, maybe somehow extract to meteor

File: src/Lykke.Snow.PriceAlerts.Domain/Services/IMeteorService.cs (hunk 3 above)
Comment (gponomarev-lykke) [reply src/Lykke.Snow.PriceAlerts.Domain/Services/IMeteorService.cs:6]: Yep, will fix

**Pull request merged**